docs/results.md

```diff
@@ -0,0 +1,376 @@
+
+```
+
+Each Scale test result directory contains directories and files related to the overall Scale test run as well as each iteration of the scenario.
+For example:
+```
+results/scale/PERF_SCALE_1572446292
+├── PERF_SCALE_1572446292.csv
+├── PERF_SCALE_1572446292.csv.html
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+├── Scale_1572446292_3_800
+├── beaker_environment.txt
+├── json
+├── log
+├── pe_tune_current.txt
+└── puppet-metrics-collector
+```
+
+## Scale test report
+The Scale test results contains a CSV file with measurements extracted from the Gatling JSON data for each iteration.
+This report is useful for comparing the performance impact of additional nodes for each iteration.
+It is also useful for comparing the outcome of multiple Scale test runs.
+
+In the example above this file is `PERF_SCALE_1572446292.csv`; 
+as with other CSV files in the results it has been converted to an HTML document (`PERF_SCALE_1572446292.csv.html`).
+
+## Beaker environment settings
+The `beaker_environment.txt` file contains the environment variables for the test run.
+
+## Iterations
+Each iteration of a Scale test run is actually a separate Performance test run and will create a separate result in the `results/perf` directory.
+The Scale results are then constructed by selectively copying specific directories and files from the results for each iteration of the run in order to provide a cohesive overview of the entire Scale test run and to overcome the somewhat inconvenient organizational structure of the Performance results.
+
+A separate directory is created within the Scale test results for each iteration, labeled with the following pattern:
+```
+Scale_{timestamp}_{iteration}_{number_of_agents}
+```
+
+This pattern makes it easy to tell how many iterations were run and how many agents were run per iteration.
+For example, the directory listing above is from a sample test with three iterations:
+```
+results/scale/PERF_SCALE_1572446292
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+└── Scale_1572446292_3_800
+```
+
+### Iteration sub-directories
+The sub-directory created for each Scale test iteration contains directories and files selectively copied from the corresponding Performance results for that iteration.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600
+├── 20191030T143909Z-20191030T151209Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── master
+├── metric
+├── puppet-metrics-collector
+├── puppetserver.average.csv
+├── puppetserver.average.csv.html
+├── puppetserver.csv
+├── puppetserver.csv.html
+└── start_epoch
+```
+
+#### Host directories
+The first obvious difference between the Scale results and the Performance results is that the 'master' and 'metric' nodes are identified by their role rather than hostname.
+
+##### Master
+The 'master' directory contains the same files found in the corresponding Performance results directory.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.